docs/references/strategies/README.md

```diff
@@ -105,3 +103,39 @@
 - [carthage](platforms/ios/carthage.md)
 - [cocoapods](platforms/ios/cocoapods.md)
 - [swiftPM](platforms/ios/swift.md)
+
+## Strategies by type
+
+Languages supported by FOSSA CLI can have multiple strategies for detecting dependencies, one primary strategy that yields ideal results and zero or more fallback strategies. Within this list of strategies, we have the concept of *static* and *dynamic* strategies. Static strategies parse files to find a dependency graph (example: parse a `package-lock.json` file). Dynamic strategies are required when analyzing package managers that do not offer complete lockfiles, such as Gradle or Golang. Dynamic strategies require a working build environment to operate in.
+
+> If the FOSSA CLI uses a fallback strategy and does not detect ideal results, you will see a "Warning" in the output when running `fossa analyze`.
+
+\* - denotes the primary strategy (all other strategies are fallbacks)
```

Comment:
[optional] I recommend having a third column, "Primary", which then contains one of the literals "Dynamic" or "Static". I think this makes it more clear.